templates/style.php

```diff
@@ -94,7 +94,7 @@
 blockquote {
 	color: <?php echo sanitize_hex_color( $text_color ); ?>;
 	background: rgba(127,127,127,.125);
-	border-left: 2px solid <?php echo sanitize_hex_color( $link_color ); ?>;
+	border-<?php echo is_rtl() ? 'right' : 'left'; ?>: 2px solid <?php echo sanitize_hex_color( $link_color ); ?>;
```

Comment:
The Travis build error should be resolved by doing this:

```suggestion
	border-<?php echo is_rtl() ? 'right' : 'left'; ?>: 2px solid <?php echo sanitize_hex_color( $link_color ); // WPCS: XSS OK ?>;
```